repos/fdbt-netex-output/package.json

```diff
@@ -23,7 +23,7 @@
         "aws-param-store": "^3.2.0",
         "aws-sdk": "^2.938.0",
         "fdbt-types": "file:../fdbt-types",
-        "libxslt": "^0.10.0",
+        "libxslt": "^0.10.1",
```

Comment:
Can this be moved to devDependencies now?

Reply:
Yep, moved.